render/webgl/src/lib.rs

```diff
@@ -1088,6 +1089,14 @@ impl RenderBackend for WebGlRenderBackend {
     ) -> Result<(), BitmapError> {
         Err(BitmapError::Unimplemented)
     }
+
+    fn debug_info(&self) -> Cow<'static, str> {
+        if self.gl2.is_some() {
```

Comment:
We actually can get debug info for WebGL with `gl.getParameter` constants `gl.RENDERER`, `gl.VENDOR`, and `gl.VERSION` which I believe are documented here:

https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/Constants

These, obviously, get normalized to less-fingerprinty values (for example, Firefox misreports my 1080ti as a 980) but might still be useful for error reporting.

Reply:
Added!